dill/_dill.py

```diff
@@ -15,7 +15,7 @@
 Test against "all" python types (Std. Lib. CH 1-15 @ 2.7) by mmckerns.
 Test against CH16+ Std. Lib. ... TBD.
 """
-__all__ = ['dump','dumps','load','loads','dump_session','load_session',
+__all__ = ['dump','dumps','load','loads',
```

Comment:
Removing these is probably fine, as it can be assumed that no on is importing `dump_session` from `dill._dill`.